proposals/0007-access-policy.md

```diff
@@ -0,0 +1,162 @@
+
+[RFCs](https://www.rust-lang.org/governance) have sub-teams that review and
+triage new proposed changes. This is a possibility, but would require more
+granular labeling of different SIMDs.
+
+[PEP](https://peps.python.org/pep-0013/) has an election of a council that
+reviews and triages new proposed changes. The PEP approach may result in an
+increase in governance overhead.
+
+## New Terminology
+
+Core Contributor - Individuals with Write access to SIMDs that can assist in
+triaging different SIMDs.
+
+## Detailed Design
+
+### Level 1 - Triage Access
+
+Requirements: One voucher from any user with level 2 or above access.
+
+Contributors with triage access will have the associated [triage Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Contributor role in the Solana Tech
+Discord. This is the lowest level of access.
+
+### Level 2 - Write Access
+
+Requirements: One voucher from anyone with level 3 access or two vouchers from
+anyone from level 2. Applicants must have level 1 access to apply for level 2.
+
+Contributors with write access will have the associated [write Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Core Contributor role in the Solana Tech
+Discord.
+
+Most notably these permissions allow the contributor to approve and merge pull
+requests.
+
+### Level 3 - Maintain Access
+
+Requirements: One voucher from anyone with level 3 access.
+
+Contributors will have the associated [maintain Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Contributor role in the Solana Tech
+Discord.
+
+Users with maintain access are responsible for managing the SIMD repository and
+granting user access based on the above requirements.
+
+### Vouching Process 
+
+To receive a level of access as defined in SIMD-0007, follow these steps:
+
+1. Open an issue on the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository with a title in the following format: "Request Access (Level [X])
+for [Username]".
+2. Gather your vouchers to add a comment on the issue expressing their support.
+3. Once the issue has received enough support, you will be granted the
```

Comment:
How about pinging a "@SIMD-Maintainers" or something github team at this step too, to try to help summon a Maintainer.